_scripts/migrations/0015_discussion_access_and_invites.sql

```diff
@@ -0,0 +1,98 @@
+ALTER TABLE discussions ADD COLUMN IF NOT EXISTS public_access boolean default true not null;
+
+CREATE TABLE IF NOT EXISTS discussion_flair_access (
+    discussion_id varchar(36) not null,
+    flair_template_id varchar(36) not null,
+    created_at timestamp with time zone default current_timestamp not null,
+    deleted_at timestamp with time zone,
+    PRIMARY KEY(discussion_id, flair_template_id)
+);
+
+CREATE TABLE IF NOT EXISTS discussion_user_access (
+    discussion_id varchar(36) not null,
+    user_id varchar(36) not null,
+    created_at timestamp with time zone default current_timestamp not null,
+    deleted_at timestamp with time zone,
+    PRIMARY KEY(user_id, discussion_id)
+);
+
+CREATE TABLE IF NOT EXISTS discussion_invite_link_access (
+    discussion_id varchar(36) PRIMARY KEY,
+    vip_invite_url text,
+    invite_url text,
```

Comment:
Realistically this could just be some ID (or UUID) rather than having to store the entire URL.

Reply:
updated